common/src/service-connectors/notification/schemas/SendNotification.schema.ts

```diff
@@ -0,0 +1,73 @@
+import {z} from "zod";
+
+import {
+    SendDirectMessage_Request,
+    SendGuildTextMessage_Request,
+    SendWebhookMessage_Request,
+} from "../../bot";
+
+export enum NotificationType {
+    RANKDOWN = "RANKDOWN",
```

Comment:
Don't we also have notifications for scrims?

Reply:
Those are handled by events.